source/releasenotes/2025-05-08-workspace-traffic-insights.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Site traffic now visible from workspace site list
+published_date: "2025-05-08"
+categories: [new-feature, user-interface]
+---
+We've added a new **Traffic** column to the site list of workspaces, enabling clearer visibility into traffic usage across all sites. This data resets at the end of your billing period. 
```

Comment:
Somewhere I feel we need to specify whether this reflects site visitors or pages served, that's not clear at the moment. @carly-armstrong can you confirm which we're seeing in this context?